src/components/app-header/indexWithSession.tsx

```
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import AppHeader from  './index'
 import { auth } from "@/auth"
+import {prisma} from "@/prisma";
```

ESLint

[error] 'prisma' is defined but never used.

(@typescript-eslint/no-unused-vars)